code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -138,6 +138,10 @@
 		cat_ears.color = human_for_preview.hair_color
 		human_for_preview.update_body()
 
+/datum/species/human/felinid/get_physical_attributes()
+	return "Felinids are very similar to humans in almost all respects, with their biggest differences being the ability to lick their wounds, \
+		and an increased sensitivity to noise, which is often detrimental."
```

Comment:
```suggestion
		and an increased sensitivity to noise, which is often detrimental. They are also rather fond of oranges."
```

you can ignore this if you want to but it's pretty funny (and enshrined in code too)

Reply:
demented